src/backend/platforms/index.ts

```diff
@@ -0,0 +1,77 @@
+import { databaseUpsert } from "../database/database-wrapper";
+import { betfair } from "./betfair";
+import { fantasyscotus } from "./fantasyscotus";
+import { foretold } from "./foretold";
+import { goodjudgment } from "./goodjudgment";
+import { goodjudmentopen } from "./goodjudmentopen";
+import { infer } from "./infer";
+import { kalshi } from "./kalshi";
+import { manifoldmarkets } from "./manifoldmarkets";
+import { metaculus } from "./metaculus";
+import { polymarket } from "./polymarket";
+import { predictit } from "./predictit";
+import { rootclaim } from "./rootclaim";
+import { smarkets } from "./smarkets";
+import { wildeford } from "./wildeford";
+
+export interface Forecast {
+  id: string;
+  title: string;
+  url: string;
+  description: string;
+  platform: string;
+  options: any[];
+  timestamp: string;
+  qualityindicators: any;
+  extra?: any;
```

Comment:
Forecasts taken from the `Database of existential risk estimates` also have an optional author property. Though they seem to be missing from the current webpage??

Reply:
Made an issue https://github.com/QURIresearch/metaforecast/issues/39

Reply:
> Forecasts taken from the Database of existential risk estimates also have an optional author property. 

process-forecasts-from-xrisk returns `moreoriginsdata: { author: datum.author }`, but I can't find it being used anywhere. Maybe it should use `extra` instead of `moreoriginsdata`?..

I based Platform type on database structure, there are 10 columns and I assumed anything else won't be stored in the DB and will be lost. But I haven't looked at each fetcher in details, have I missed something?

Re: x-risks being gone, I'll investigate and comment on #39.